modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/suspended/concurrent/Fx.kt

```diff
@@ -0,0 +1,16 @@
+package arrow.effects.typeclasses.suspended.concurrent
+
+import arrow.Kind
+import arrow.core.Tuple2
+import arrow.effects.typeclasses.Concurrent
+import arrow.effects.typeclasses.ConcurrentCancellableContinuation
+import arrow.effects.typeclasses.Disposable
+
+interface Fx<F> {
```

Comment:
Why is every Fx called Fx even if they're for different typeclasses? Are we relying on IJ to pull the right one?

Reply:
I guess where strongly relying on imports

Reply:
See https://github.com/arrow-kt/arrow/pull/1263#discussion_r